portals/publisher/source/src/app/components/Apis/Details/Endpoints/Prototype/MockedOASOperation.jsx

```diff
@@ -0,0 +1,222 @@
+                            SelectProps={{
+                                autoWidth: true,
+                                IconComponent: ArrowDropDown,
+                            }}
+                            className={classes.dropdown}
+                            label='Media Type'
+                            value={selectedMediaType}
+                            onChange={({ target: { value } }) => {
+                                setSelectedMediaType(value)
+                            }}
+                            helperText='Controls Accept header'
+                            margin='dense'
+                            variant='outlined'
+                        >
+                            {Object.keys(content)
+                                .map((mediaType,) => (
+                                    <MenuItem
+                                        key={mediaType}
+                                        value={mediaType}>
+                                        {mediaType}
+                                    </MenuItem>
+                                ))
+                            }
+                        </TextField>
+                        {selectedExampleType &&
+                            <TextField
+                                select
+                                SelectProps={{
+                                    autoWidth: true,
+                                    IconComponent: ArrowDropDown,
+                                }}
+                                className={classes.dropdown}
+                                label='Examples'
+                                value={selectedExampleType}
+                                onChange={({ target: { value } }) => {
+                                    setSelectedExampleType(value)
+                                }}
+                                margin='dense'
+                                variant='outlined'
+                            >
+                                {Object.keys(content[selectedMediaType].examples)
+                                    .map((exampleType,) => (
+                                        <MenuItem
+                                            key={exampleType}
+                                            value={exampleType}>
+                                            {exampleType}
+                                        </MenuItem>
+                                    ))
+                                }
+                            </TextField>}
+                        <Typography variant='body1' gutterBottom>
+                            <FormattedMessage
+                                id='Apis.Details.Endpoints.Prototype.MockedOAS.Response.Example'
+                                defaultMessage='Example Value'
+                            />
+                        </Typography>
+                        <Suspense fallback={<CircularProgress />}>
+                            <MonacoEditor
```

Comment:
@ashera96 is adding a new Monaco editor package via the gateway policy feature, Once it's merged shall we switch to that package,

Reply:
ack